### PR TITLE
[G2M] Table - incorporate zpl.io/bzgyQNz

### PR DESCRIPTION
**Pull request by woozyking**
published as [0.7.0-alpha3](https://www.npmjs.com/package/@eqworks/react-labs/v/0.7.0-alpha3)

incorporate design https://zpl.io/bzgyQNz

not all filters are implemented (only default text value filter much like the global search)

### CODE REVIEW

**Comment by DoParkEQ** [comment]
Awesome! I noticed that the spacing between header columns fluctuates when data are sorted. Could we make its spacings static?

**Comment by woozyking** [comment]
> Awesome! btw I noticed that the spacing between header columns fluctuates when data are sorted. Could we make its spacings static?

this is something that shouldn't be enforced at the table level, but dependent on how `Cell` rendering function handles the length of the content of each column. so if `Cell` renders the values in a way that has a fixed width (through capping or `text-overflow: ellipsis` for instance), then it'll naturally be fixed.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@eqworks/react-labs",
   "private": false,
-  "version": "0.6.2",
+  "version": "0.7.0-alpha",
```
[the remark this answers is not in the document]
Comment (woozyking) [reply package.json:4]: it'd be under `alpha` tag so one who installs this package as 

```shell
% npm i @eqworks/react-labs@alpha
# or
% yarn add @eqworks/react-labs@alpha
```

would pick the latest version tagged under it

File: stories/table.stories.js [hunk 2]
```diff
@@ -121,6 +121,30 @@ export const initialHiddenColumns = () => (
...
+)
+
+export const disableFilters = () => (
+  <Table
+    data={provinces}
+    columns={[
+      { Header: 'New cases', accessor: 'new_cases' },
+      { Header: 'Total cases', accessor: 'total_cases' },
+      { Header: 'Rate', accessor: 'rate', filter: 'excludes', Cell: ({ value }) => `${value}%` },
+      { Header: 'Province', accessor: 'province', disableFilters: true, disableGlobalFilter: true },
```
[the remark this answers is not in the document]
Comment (woozyking) [reply stories/table.stories.js:143]: yea the rest is still a part of the Global filter (search)

i'm debating on whether to introduce a table-wise flag to disable/hide global filter (search) regardless of column config

**Pull request merged**